src/Gauge.Dotnet.csproj

```diff
@@ -23,9 +23,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Gauge.CSharp.Core" Version="0.4.1" />
+    <PackageReference Include="Gauge.CSharp.Core" Version="0.5.0-pre3" />
```

Comment:
We have to make the release of it right?

Reply:
yes, will do the release and make a push